db/migrations/postgresql/app/V2.0.0.4__table_support_requests.sql

```diff
@@ -0,0 +1,12 @@
+BEGIN;
+
+CREATE TABLE IF NOT EXISTS app.support_requests (
+    "id"            UUID PRIMARY KEY DEFAULT gen_random_uuid(),
+    "person"        JSONB,
+    "support_data"  JSONB,
+    "created_at"    TIMESTAMPTZ DEFAULT CURRENT_TIMESTAMP NOT NULL,
```

Comment:
I am not sure if it's not better to use `TIMESTAMP` and store/serve everything in UTC? 

Using the `TZ` part definitely makes it easier for the back-end but API consumers might have problems if we return the timestamps in the time zone of the server - what if the API only returns the datetime without the zone offset? Then the clients cannot reliably convert this to their own timezones.

To be safer it's probably just better to store it all in UTC 😄

Reply:
agreed

Reply:
`For timestamp with time zone, the internally stored value is always in UTC (Universal Coordinated Time, traditionally known as Greenwich Mean Time, GMT). An input value that has an explicit time zone specified is converted to UTC using the appropriate offset for that time zone. If no time zone is stated in the input string, then it is assumed to be in the time zone indicated by the system's timezone parameter, and is converted to UTC using the offset for the timezone zone.`

This is from the PostgreSQL manual (see [here](https://www.postgresql.org/docs/9.1/datatype-datetime.html)). It seems like this is the correct type and we just need to make sure that the module is actually providing the timestamp when saving data